### DISCUSSION TITLE
Unable to install Chrome extension on Brave browser; Access is denied

### DISCUSSION THREAD

**Discussion by nopapersplease** [simple-login/app#742] [Q&A] [answered]
I'm attempting to install the SimpleLogin extension from the Chrome web store on my Brave browser. I can install the extension on Chrome, but I cannot install it on Brave. I receive an "Access denied" error in the web store and the download fails with HTTP 403 return code.

Is there a fix/workaround for this? Brave is supposed to be able to run Chrome extensions; I have many such extensions running in my browser. I'd really like to have the extension working in Brave if possible.

Thanks.

**Comment by FarisZR** [comment]
i don't think this error has anything to do with simple login.
its a problem with chrome web store, or something else, but yes brave support chrome extensions.

**Comment by Theelx** [comment]
I just downloaded the extension on Brave this morning and it works like a charm, so it might be worth retrying the installation.

**Comment by nopapersplease** [comment, marked as answer]
It didn't work...until I updated Brave. Now it's working. That's the solution. Check for updates to Brave if it does not install. Thanks, everyone!
